### PR TITLE
Added resources of React

### PR DESCRIPTION
**Pull request by nucliweb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,10 @@ Some of these links are affiliate links, meaning that if you make a purchase, I
 * [react-art](https://github.com/reactjs/react-art)
 * [react-canvas](https://github.com/Flipboard/react-canvas)
 * [react-three](https://github.com/Izzimach/react-three) Three.js rendering for React
+* [react.rocks](http://react.rocks/) Pinterest-style directory of online demos with available code
+* [TodoMVC in React](http://todomvc.com/examples/react/)
+* [JSX Specification](http://facebook.github.io/jsx/)
+* [HTML to JSX Compiler](https://facebook.github.io/react/html-jsx.html)
```
Comment (ericelliott) [inline README.md]: Append to the end of this line:

```js
Currently compiles to `.createClass()`, which is likely to be deprecated in the future.
```

**Comment by ericelliott** [comment]
One tiny tweak, otherwise, top quality suggestions. Thanks!

File: README.md (hunk 1 above)
Comment (nucliweb) [inline README.md]: link to HTML to JSX Compiler removed ;)

**Comment by nucliweb** [comment]
Thanks for merge ;)

**Comment by ericelliott** [comment]
:+1:

**Pull request merged**